src/cf-forms/usage.md

```diff
@@ -213,17 +218,114 @@ Inputs should always be paired with a `label` for accessibility reasons.
 </div>
 ```
 
+The following sections demonstrate how a particular state of a checkbox
+could be forced to be shown.
+Generally this is only useful for documentation purposes.
+
+#### Hover/Focus
+
+<div class="m-form-field m-form-field__checkbox">
+    <input class="a-checkbox hover" type="checkbox" id="test_checkbox_basic_hover">
+    <label class="a-label" for="test_checkbox_basic_hover">Label</label>
+</div>
+
+```
+<div class="m-form-field m-form-field__checkbox">
+    <input class="a-checkbox hover" type="checkbox" id="test_checkbox_basic_hover">
+    <label class="a-label" for="test_checkbox_basic_hover">Label</label>
+</div>
+```
+
+#### Selected
```

Comment:
Did you forget to add the selected classes to the Less or are those coming later?

Reply:
Oh wait, you don't need one because we can set the `checked` attribute. Nice little work around there.